### PR TITLE
Adjust cluster users for addons

### PR DESCRIPTION
**Pull request by polskikiel**
**Description**

Changes proposed in this pull request:

- Created a new ClusterRole`kyma-namespace-admin` - it's like `kyma-admin` role, but without permissions to write addonsconfigurations.
- Changed roleNameRef in the permission-controller to bind a new `kyma-namespace-admin` role to customer namespaces
- Added tests for that

**Related issue(s)**

### CODE REVIEW

**Comment by polskikiel** [comment]
@piotrmsc 
I tested it and after my changes namespace-admin user has no permissions in the system namespaces as well and has admin permissions in customer namespaces. Example ss from getting all resources from `kyma-system` using namespace-admin's kubeconfig: 
image

We also have tests which ensure that namespace-admin has no permissions in the system namespaces and they pass on my PR:
```
echo "--> ${NAMESPACE_ADMIN_EMAIL} should NOT be able to get ory Access Rule in system namespace"
testPermissions "get" "rule.oathkeeper.ory.sh" "${SYSTEM_NAMESPACE}" "no"

echo "--> ${NAMESPACE_ADMIN_EMAIL} should NOT be able to create secret in system namespace"
testPermissions "create" "secret" "${SYSTEM_NAMESPACE}" "no"

echo "--> ${NAMESPACE_ADMIN_EMAIL} should NOT be able to delete system namespace"
testPermissions "delete" "namespace" "${SYSTEM_NAMESPACE}" "no"

echo "--> ${NAMESPACE_ADMIN_EMAIL} should NOT be able to create addonsconfigurations in ${SYSTEM_NAMESPACE}"
testPermissions "create" "addonsconfigurations" "${SYSTEM_NAMESPACE}" "no"

echo "--> ${NAMESPACE_ADMIN_EMAIL} should NOT be able to create serviceinstances in ${SYSTEM_NAMESPACE}"
testPermissions "create" "serviceinstances" "${SYSTEM_NAMESPACE}" "no"

echo "--> ${NAMESPACE_ADMIN_EMAIL} should NOT be able to create servicebindings in ${SYSTEM_NAMESPACE}"
testPermissions "create" "servicebindings" "${SYSTEM_NAMESPACE}" "no"
```

**Comment by polskikiel** [comment]
@piotrmsc 
I tested it and after my changes namespace-admin user has no permissions in the system namespaces as well and has admin permissions in customer namespaces. Example ss from getting all resources from `kyma-system` using namespace-admin's kubeconfig: 
image


We also have tests which ensure that namespace-admin has no permissions in the system namespaces and they pass on my PR:
```
echo "--> ${NAMESPACE_ADMIN_EMAIL} should NOT be able to get ory Access Rule in system namespace"
testPermissions "get" "rule.oathkeeper.ory.sh" "${SYSTEM_NAMESPACE}" "no"

echo "--> ${NAMESPACE_ADMIN_EMAIL} should NOT be able to create secret in system namespace"
testPermissions "create" "secret" "${SYSTEM_NAMESPACE}" "no"

echo "--> ${NAMESPACE_ADMIN_EMAIL} should NOT be able to delete system namespace"
testPermissions "delete" "namespace" "${SYSTEM_NAMESPACE}" "no"

echo "--> ${NAMESPACE_ADMIN_EMAIL} should NOT be able to create addonsconfigurations in ${SYSTEM_NAMESPACE}"
testPermissions "create" "addonsconfigurations" "${SYSTEM_NAMESPACE}" "no"

echo "--> ${NAMESPACE_ADMIN_EMAIL} should NOT be able to create serviceinstances in ${SYSTEM_NAMESPACE}"
testPermissions "create" "serviceinstances" "${SYSTEM_NAMESPACE}" "no"

echo "--> ${NAMESPACE_ADMIN_EMAIL} should NOT be able to create servicebindings in ${SYSTEM_NAMESPACE}"
testPermissions "create" "servicebindings" "${SYSTEM_NAMESPACE}" "no"
```

File: docs/security/03-05-roles-in-kyma.md [hunk 1]
```diff
@@ -19,10 +19,11 @@ The predefined roles are:
...
+| **kyma-namespace-admin-essentials** | The role that allows the user to see the Console UI and create Namespaces, built on top of the**kyma-essentials** role. Used to give the members of selected groups the ability to create Namespaces in which the Permission Controller binds them to the **kyma-namespace-admin** role. For more details, read [this](#details-permission-controller) document.  |
 | **kyma-view** | The role for listing Kubernetes and Kyma-specific resources. |
 | **kyma-edit** | The role for editing Kyma-specific resources. |
 | **kyma-developer** | The role created for developers who build implementations using Kyma. It allows you to list and edit Kubernetes and Kyma-specific resources. |
 | **kyma-admin** | The role with the highest permission level which gives access to all Kubernetes and Kyma resources and components with administrative rights. |
+| **kyma-namespace-admin** | The role with `kyma-admin` permissions without write access to AddonsConfigurations. |
```
Comment (klaudiagrz) [inline docs/security/03-05-roles-in-kyma.md]:
```suggestion
| **kyma-namespace-admin** | The role that has the same rights as the **kyma-admin** role, except for the write access to [AddonsConfigurations](https://kyma-project.io/docs/master/components/helm-broker#custom-resource-addons-configuration). |
```

File: docs/security/08-02-manage-static-users.md [hunk 2]
```diff
@@ -43,6 +43,7 @@ A newly created static user has no access to any resources of the cluster as the
 By default, Kyma comes with the following ClusterRoles:
 
 - **kyma-admin**: gives full admin access to the entire cluster
+- **kyma-namespace-admin**: gives an `kyma-admin` access without write permissions for AddonsConfigurations.
```
Comment (klaudiagrz) [inline docs/security/08-02-manage-static-users.md]:
```suggestion
- **kyma-namespace-admin**: gives full admin access except for the write access to AddonsConfigurations
```

File: docs/security/08-02-manage-static-users.md [hunk 3]
```diff
@@ -43,6 +43,7 @@ A newly created static user has no access to any resources of the cluster as the
 By default, Kyma comes with the following ClusterRoles:
 
 - **kyma-admin**: gives full admin access to the entire cluster
+- **kyma-namespace-admin**: gives full admin access except for the write access to AddonsConfigurations
```
Comment (klaudiagrz) [inline docs/security/08-02-manage-static-users.md]:
```suggestion
- **kyma-namespace-admin**: gives full admin access except for the write access to [AddonsConfigurations](https://kyma-project.io/docs/master/components/helm-broker#custom-resource-addons-configuration)
```

**Pull request merged**